docker-compose.yml

```diff
@@ -25,13 +37,17 @@ services:
     # Overwrite any env var defined in .env file (if required)
     environment:
       - MONGO_HOST=mongodb://mongo/express-mongoose-es6-rest-api-development
```

Comment:
I'm fine with doing it this way, but we can also just add the env to the up command `--env-file` I believe.

Reply:
I moved the env variables to the .env file on my last commit.